src/test/java/com/fasterxml/jackson/databind/cfg/CacheProviderTest.java

```diff
@@ -0,0 +1,57 @@
+package com.fasterxml.jackson.databind.cfg;
+
+import com.fasterxml.jackson.databind.*;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.databind.util.LookupCache;
+import org.junit.Test;
+
+import java.util.Map;
+
+public class CacheProviderTest extends BaseMapTest {
+    
+    static class RandomBean {
+        public int point;
+    }
+
+    static class SimpleExampleCache implements LookupCache<JavaType, JsonDeserializer<Object>> {
```

Comment:
Maybe `SimpleTestCache` -- just make it even more obvious when auto-completing etc.